### PR TITLE
Rerender app component on resize

### PR DESCRIPTION
**Pull request by dgliwka**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -109,6 +109,21 @@ export default class App extends Component<{}, { chosenNode: NodeID, value: stri
 
     }
 
+    handleResize()
+    {
+        this.setState(prevState => prevState);
```
Comment (allgreed) [inline src/App.tsx:114]: how about `forceUpdate`? Seams like a more appropriate solution ;)

File: src/App.tsx [hunk 2]
```diff
@@ -109,6 +109,21 @@ export default class App extends Component<{}, { chosenNode: NodeID, value: stri
...
+    }
+
+    componentDidMount()
+    {
+        window.addEventListener("resize", this.handleResize.bind(this));
+    }
+
+    componentWillUnmount()
+    {
+        window.removeEventListener("resize", this.handleResize.bind(this));
```
Comment (allgreed) [inline src/App.tsx:124]: Will the removal work? Seams like they're two distinct function objects.

Comment (dgliwka) [reply src/App.tsx:124]: IDK, but they told me so on the internet.

Comment (allgreed) [reply src/App.tsx:124]: It's a very good style btw

**Pull request merged**